google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Acl.java

```diff
@@ -568,9 +570,82 @@ Access toPb() {
     }
   }
 
+  /** Expr represents the conditional information related to dataset access policies. */
+  public static final class Expr implements Serializable {
```

Comment:
An addition of:
  private static final long serialVersionUID = somevalueL;
to be consistent.

Reply:
Fixed.